<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1093</width>
    <height>847</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stock</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="PlotWidget" name="graphWidget" native="true">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>30</y>
      <width>741</width>
      <height>561</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>30</y>
      <width>271</width>
      <height>561</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>買</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>價格</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="buy_btn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>700</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>確定</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="buy_input">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>700</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="buy_label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>650</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>我要買</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="sell_input">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>700</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="sell_label">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>650</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>我要賣</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sell_btn">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>700</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>確定</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1093</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
